File: Estrutura_Repetição/Python/Ex1116.py
n = int(input())

for i in range(0, n):
    x, y = map(int, input().split())

    if y == 0:
        print("divisao impossivel")
    elif x == 0:
        print(0.0)
    else:
        valor: float = x / y
        print(f"{valor:.1f}")

File: Estrutura_Repetição/JavaScript/Ex1116.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var prompt = function(texto) { return lines.shift();};

    var N=parseInt(prompt(""));
    
    for(i=0;i<N;i++){
        var [X,Y]=prompt("").split(" ").map(Number)
        
        var div=X/Y
        
        if(Y===0){
            console.log("divisao impossivel")
        } else{
            console.log(div.toFixed(1))
   }
}